action.yml

```diff
@@ -11,6 +11,12 @@ inputs:
   slackWebhookURL:
     description: |
       A Slack incoming webhook URL that you want findings posted to.
+  semgrepAppToken:
+    description: |
+      Access token to post results to the semgrep app.
```

Comment:
In some magical future this should be pulled out of GitHub secretes

Reply:
That's the intention. In the workflow config the user has to explicitly map their github secret to our input argument. There's no way for us to access a github secret directly.